app/Modules/Endorsement/Solo/SoloEndorsementService.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Modules\Endorsement\Solo;
+
+use App\Modules\Position\Position;
+use App\User;
+use Carbon\Carbon;
+
+class SoloEndorsementService
+{
+    private const DEFAULT_DURATION_IN_DAYS = 30;
```

Comment:
Could make this a .env thing so it can be changed if needed?

Reply:
Personally, I think the current solution is ok? It's unlikely we would want in altered that often and there wouldn't be a reason for it to differ between environments?

Reply:
I'd keep it as is since it is business logic